e-store/e-store/ScenesScreen/Categories/CategoriesViewModel.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  CategoriesViewModel.swift
+//  e-store
+//
+//  Created by Zeyad on 07/05/2023.
+//
+
+import Foundation
+import UIKit
```

Comment:
can't import UIKit in view model

Reply:
i must import it so i can use the uiimage class